.github/workflows/lint.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: "Lint"
-on: # yamllint disable-line rule:truthy
+on:  # yamllint disable-line rule:truthy
```

Comment:
Why this change and below?

Reply:
`yamllint` wants two spaces before an inline comment and appeared to be failing the lint check as a result.

Reply:
Blech